tests/integration/release-integration-test.sh

```diff
@@ -0,0 +1,238 @@
+#! /bin/bash
```

Comment:
Is this test to make sure some upgrade works like xoph suggested? If so, do we need all the tests or can we reduce the amount of tests run?
Also, what do you think of writing a comment at the start of this/each integration test highlighting why it is needed in addition to the base case?